src/cli/mod.rs

```diff
@@ -13,9 +13,10 @@ use crate::tool_spec::ToolSpec;
 use crate::tool_storage::{InstalledToolsCache, ToolStorage};
 use crate::trust::{TrustCache, TrustMode};
 
-#[derive(Debug, StructOpt)]
+#[derive(Debug, Parser)]
+#[clap(version)]
```

Comment:
```suggestion
#[clap(author, version, about)]
```

I don't know if clap does these things by default like structopt does?

Reply:
These aren't on by default but they didn't appear to be on before. Should I add an you as an author in the `Cargo.toml` as well then?

Reply:
Yeah, good call. Thank you!